docs/cql/ddl.rst

```diff
@@ -111,7 +111,7 @@ For example:
 .. code-block:: cql
 
    CREATE KEYSPACE Excelsior
-   WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 3};
+   WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor' : 3};
```

Comment:
I am not sure about it, but it looks to me like `SimpleStrategy` was used here only as a replication strategy different from the one used above. If we still don't like SimpleStrategy, perhaps we could replace it with EverywhereStrategy instead of NetworkTopology?